recipes/msgpack-cxx/all/conanfile.py

```diff
@@ -23,18 +32,14 @@ def _source_subfolder(self):
     def _build_subfolder(self):
         return "build_subfolder"
 
-    def requirements(self):
-        self.requires("boost/1.78.0")
+    def configure_options(self):
+        # No boost was added in 4.1.0
+        if tools.Version(self.version) < "4.1.0":
+            del self.options.use_boost
 
-    def validate(self):
-        if self.options["boost"].header_only == True:
-            raise ConanInvalidConfiguration("{} requires boost with header_only = False".format(self.name))
-
-        if self.options["boost"].without_chrono == True or \
```

Comment:
are those components no longer required?

Reply:
they were not required, it was a mistake I think